### ISSUE TITLE
Feature request: keep focus on the log record when toggling levels

### ISSUE THREAD

**Issue by dimapu** [Busimus/cutelog#19]
First, I wanted to thank the authors for the nice piece of software. 

I have the following feature request, which would help studying the logs. 

#### Use case
Let's say, I have `INFO` (not too many) and `DEBUG` (a lot!) messages. When disabling display of `DEBUG` messages in the Levels panel, I can conveniently browse through the `INFO` messages and find the one I am interested in. I click on it in the entry list to select it. Then I click on a `DEBUG` tick box to start showing more detailed log.

#### What happens now
The list of log entries is scrolled and the selected `INFO` entry moves out of the screen. It takes effort to find it by its time, for example.

#### Desired behavior
The selected entry stays on the screen, and I can immediately explore `DEBUG` entries just before and just after the selected one.

Hope this is clear and you will find it useful too.
Regards, 
Dima.

**Comment by ankansankan** [comment]
Yes, this is really necessary when working with this tool? Any patch available?

**Comment by potatopplking** [comment]
A friend of mine faced the same issue. I made a fix in PR #50. With it the selected item is kept in focus.
